test/e2e/kubernetes/pod/pod.go

```diff
@@ -279,7 +279,7 @@ func RunWindowsPod(image, name, namespace, command string, printOutput bool, sle
 type podRunnerCmd func(string, string, string, string, bool, time.Duration, time.Duration, time.Duration) (*Pod, error)
 
 // RunCommandMultipleTimes runs the same command 'desiredAttempts' times
-func RunCommandMultipleTimes(podRunnerCmd podRunnerCmd, image, name, command string, desiredAttempts int, sleep, commandTimeout, timeout time.Duration) (int, error) {
+func RunCommandMultipleTimes(podRunnerCmd podRunnerCmd, image, name, command string, desiredAttempts int, sleep, commandTimeout, timeout time.Duration, StabilityIterationsSuccessRate float32) (int, error) {
```

Comment:
This particular func only need to know if `StabilityIterationsSuccessRate == 1.0`. Hence, we should pass a bool parameter and name it accordingly. Maybe `exitOnFailure`?